### ISSUE TITLE
Improvements to CV browser

### ISSUE THREAD

**Issue by tgherzog** [worldbank/ESG_gaps_research#32]
@tonyfujs - I wanted to see if there's the possibility to make a few fixes to the CV browser tool you added to what's currently section 4.3:

1. Limit the maximum year to 2018 (this will be true for all charts in this section)
2. For some reason the chart disappears for me when the "Maximum CV" is set to 0.7 or 0.3 (0.6 and 0.8 are fine though)
3. Make the CV slider move in increments of 0.05 instead of 0.1 for more precision
4. In the column chart can the indicator width be fixed? It seems to vary so that the total width of the chart is constant. It's actually important to see that as you adjust the sliders the total number of indicators for which you have values can change, as shown below
5. Is it possible to add a legend? In the python version the legend also lets you toggle each series on and off.

![image](https://user-images.githubusercontent.com/1315823/84477946-82d32380-ac5e-11ea-9034-31bd390fe992.png)

![image](https://user-images.githubusercontent.com/1315823/84477996-92526c80-ac5e-11ea-9997-93f156765887.png)

**Comment by tonyfujs** [comment]
@tgherzog Yes, the current version of the chart is quite rough. Let me see what I can do to address the specific points you raised. Will get back to you early next week on this.

**Comment by tonyfujs** [comment]
Hi @tgherzog I think I addressed most of your comments in the updated app. Let me know.

**Comment by tgherzog** [comment]
Hi @tonyfujs - would it be possible in the legend to show AUC as well as indicator count as shown in the screen grab below? These are ideas that are developed in the paper and it would be good to show them to the user.

![image](https://user-images.githubusercontent.com/1315823/87116355-9369c000-c243-11ea-80e6-b22338f96348.png)

AUC is "area under curve" and in this case is defined as the sum of the bar values divided by the number of indicators. Or another way, the number of MRVs relative to the selected year divided by nCountries * nSeries.

N is simply the number of indicators that have % coverage > 0.

You can also refer to my python notebook if helpful:

https://github.com/worldbank/ESG_gaps_research/blob/master/python/coverage-analysis.ipynb

**Comment by tonyfujs** [comment]
Hi @tgherzog 
Sure. Let me take a look at this.

**Comment by tonyfujs** [comment]
@tgherzog I just updated the app. https://datanalytics.worldbank.org/esg_imputation/
Let me know when you have a few minutes to clear out a couple of questions I have.
Thanks!

**Comment by tgherzog** [comment]
@tonyfujs I just took a look. Visually, looks good, but I'm seeing different results for the same input parameters comparing yours to mine in the python notebook above in both the chart and the stats in the legend (I added some code to the notebook so you could see the raw output in github even if you can't run it):

````

Year/Max CV/Years to Impute:                 2018/0.5/1
Total Indicators:                            115
Total Economies:                             217
Total Observations:                          24955

Actual Observations (before/after):          7659/8333
AUC (before/after):                          30.7/33.4

Indicators with MRV>=2018 (before/after):    54/75
Improved Indicators (previously 0%/all):     21/43
Indicators with 50% coverage (before/after): 38/38
````

**Comment by tonyfujs** [comment]
Thanks @tgherzog Will look at this.

**Comment by tonyfujs** [comment]
Hi @tgherzog 
Everything should be good now. My code was relying on an outdated ESG_cv.feather file: I streamlined this so it now depends on the exact same files used in the variability chapter.
